Packs/CommonScripts/Scripts/DumpJSON/DumpJSON.py

```diff
@@ -3,8 +3,13 @@
 import demistomock as demisto  # noqa: F401
 from CommonServerPython import *  # noqa: F401
 
-JSON_CONTEXT_KEY = "JsonStr"
-key = demisto.args()['key']
-objStr = json.dumps(demisto.get(demisto.context(), key))
-demisto.setContext(JSON_CONTEXT_KEY, objStr)
-demisto.results(objStr)
+
+def main():
+    key = demisto.args()['key']
+    obj_str = json.dumps(demisto.get(demisto.context(), key))
+    demisto.setContext('JsonStr', obj_str)
+    demisto.results(obj_str)
```

Comment:
as long as we visit this script...
```suggestion
    return_results(obj_str)
```